packages/core/admin/server/bootstrap.js

```diff
@@ -85,7 +85,6 @@ module.exports = async () => {
   await roleService.resetSuperAdminPermissions();
   await roleService.displayWarningIfNoSuperAdmin();
 
-  await permissionService.ensureBoundPermissionsInDatabase();
```

Comment:
Do you have a way to reset default roles' permissions to their original state?
Or is it something left out intentionally?